docs/running-jobs/interactive-jobs.md

```diff
@@ -43,6 +45,41 @@ Expand the SSH tab on the left side of the application and click X11.
 
 In the X11 Menu check the "Enable X11 Forwarding" checkbox and type "localhost:0" in the X display location field.  Clicking open will open a terminal window where you can login.
 
+##### Command Prompt, Powershell
+
+This tutorial assumes that you are already set up with OpenSSH for Windows. [Here](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui) is a guide for setting up OpenSSH for Windows. 
```

Comment:
I didn't have OpenSSH server set up, so I followed the instructions you linked. The last step is to start the OpenSSH server, but it wasn't clear to me if I needed to have it running for your instructions to work. Can you clarify this? e.g., "Note that OpenSSH Server does not have to be running for you to proceed."